requirements.txt

```diff
@@ -7,3 +7,4 @@ matplotlib
 jsmin
 jsonschema
 netCDF4
+gdal
```

Comment:
why adding gdal here? this file should include only the base dependencies. Also, are you sure that pip is the right way to install GDAL? 

For the tests, you need to list the dependency in the `ci/ci_test_env.yml` file, see [946c97f](https://github.com/pySTEPS/pysteps/pull/284/commits/946c97fca0ef9aa031323cfdcf051d5ba30e7ac8)

Also, you should instead list GDAL as optional dependency for the geotiff FMI importer in the [user guide](https://github.com/pySTEPS/pysteps/blob/master/doc/source/user_guide/install_pysteps.rst).